src/plugins/data/public/plugin.ts

```diff
@@ -109,22 +108,7 @@ export class DataPublicPlugin
   ): DataPublicPluginSetup {
     const startServices = createStartServicesGetter(core.getStartServices);
 
-    expressions.registerFunction(indexPatternLoad);
-    expressions.registerFunction(
-      getEsaggs({
-        getStartDependencies: async () => {
-          const [, , self] = await core.getStartServices();
-          const { fieldFormats, indexPatterns, query, search } = self;
-          return {
-            addFilters: query.filterManager.addFilters.bind(query.filterManager),
-            aggs: search.aggs,
-            deserializeFieldFormat: fieldFormats.deserialize.bind(fieldFormats),
-            indexPatterns,
-            searchSource: search.searchSource,
-          };
-        },
-      })
-    );
```

Comment:
I moved esaggs function registration to the search service.

Did the same for indexPatternLoad in the index pattern service on the server, however kept it here on the client as there's no separate index pattern "service" that gets called in `setup`